### ISSUE TITLE
confusion about the choice of date range

### ISSUE THREAD

**Issue by Hessen525** [maechler/nnilm#16]
Hi Maechler ,
Thanks for nice code. I am a new in nilm and have confusion about the choice of data range in NILM Area, could you please give me some advices if possible?

I want to compare algorithms, what is the standard for choosing date range? As I see in some paper, like Neural nilm, Sequence-to-point learning with neural networks for non-intrusive load monitoring, Sliding Window Approach for Online Energy Disaggregation Using Artificial Neural Networks etc, where author train and test specified houses in ukdale or redd without detail about begin-end(window) date. 
If I want to compare my model with existing model described in paper, need i use same date range(windows) on same dataset?   

Thank you so much,
 Appreciate!

**Comment by maechler** [comment]
Hi @Hessen525,

It would be best if you could use exactly the same split as in the papers you want to compare yourself with as the split can make a difference. Imagine an extreme split where you would only have one single test sample, if you could predict this single instance correctly you would have an accuracy of 100%, if you failed your accuracy would be 0%. 

If this is however not possible because the papers are not mentioning the details, then I would probably go for an 80% train and 20% test data split. But this really depends on your data and it is indeed difficult to compare between different datasets like REDD, UK-DALE or ECO.

In the end you just need to document clearly what you did and that probably the results might differ if you knew exactly what test / train split was used in the other papers.

Does this help you / answer your question?

**Comment by Hessen525** [comment]
Hi maechler,
Thank you so much, it is helpful.

**Comment by maechler** [comment]
Great, I'm glad I could help.
